packages/vite/src/node/build.ts

```diff
@@ -133,6 +135,20 @@ export interface BuildOptions {
    * @default minify
    */
   cssMinify?: boolean
+  /**
+   * @experimental
+   * Use Lightning CSS to minify CSS. This should be installed as a peer
+   * dependency. Targets is automatically converted from esbuild (css)target, but
+   * can be provided for more granular control.
+   * @default esbuild
+   */
+  cssMinifier?:
+    | { minifier: 'esbuild' }
+    | {
+        minifier: 'lightningcss'
+        targets?: LightningCSS['Targets']
+        drafts?: LightningCSS['Drafts']
```

Comment:
Should we follow the same pattern used for the transformer here? So a `lightingcss` options here
https://github.com/vitejs/vite/pull/12807/files#diff-2cfbd4f4d8c32727cd8e1a561cffbde0b384a3ce0789340440e144f9d64c10f6R105